start.R

```diff
@@ -220,14 +220,14 @@ if (any(c("--testOneRegi", "--debug", "--quick") %in% flags) & "--restart" %in%
 
 # Check if dependencies for a model run are fulfilled
 if (packageVersion("lucode2") >= "0.34.0") {
-  lucode2::checkDeps(action = "ask")
+  piamenv::checkDeps(action = "ask")
```

Comment:
here, you need to check `piamenv` instead of `lucode2`, right? Or does `lucode2 >= 0.34.0` already imply `piamenv`?

Reply:
Oh good point, will do that